### PR TITLE
sessions_controller - remove deprecated route update swagger

### PR DESCRIPTION
**Pull request by saneshark**

### CODE REVIEW

**Comment by saneshark** [comment]
@jkassemi I am tagging you just in case for some reason we had some rewrite rules or something in nginx configuration for this route I'm removing / previously deprecated no longer used by F/E.

File: spec/controllers/v0/sessions_controller_spec.rb [hunk 1]
```diff
@@ -178,53 +178,29 @@
...
-        expect(response).to have_http_status(401)
-      end
+    it 'returns does not allow fetching the identity proof url' do
+      get :identity_proof
+      expect(response).to have_http_status(401)
+    end
 
-      it 'creates the saml authn request with LOA 3 if supplied level=3' do
-        get :new, level: 3
-        expect(SAML::AuthnRequestHelper.new(response).loa3?).to eq(true)
```
[the remark this answers is not in the document]
Comment (saneshark) [reply spec/controllers/v0/sessions_controller_spec.rb]: 👍 removing

File: config/routes.rb [hunk 2]
```diff
@@ -19,7 +19,7 @@
       end
     end
 
-    resource :sessions, only: [:new, :destroy] do
+    resource :sessions, only: :destroy do
```
[the remark this answers is not in the document]
Comment (U-DON) [reply config/routes.rb]: No instances of `/sessions/new`  anymore in the FE.

File: spec/controllers/v0/sessions_controller_spec.rb [hunk 3]
```diff
@@ -178,53 +178,29 @@
...
+      get :authn_urls
+      expect(response).to have_http_status(200)
+      expect(JSON.parse(response.body).keys).to eq %w(mhv dslogon idme)
+    end
 
-      it 'does not allow fetching the multifactor url' do
-        get :multifactor
-        expect(response).to have_http_status(401)
-      end
+    it 'returns does not allow fetching the identity proof url' do
```
Comment (U-DON) [inline spec/controllers/v0/sessions_controller_spec.rb]: I don't think the 'returns' is supposed to be there.

Comment (saneshark) [reply spec/controllers/v0/sessions_controller_spec.rb]: good catch.

**Pull request merged**